Modules/_datetimemodule.c

```diff
@@ -4663,7 +4663,10 @@ static PyMethodDef time_methods[] = {
     {"isoformat",   (PyCFunction)(void(*)(void))time_isoformat,        METH_VARARGS | METH_KEYWORDS,
      PyDoc_STR("Return string in ISO 8601 format, [HH[:MM[:SS[.mmm[uuu]]]]]"
                "[+HH:MM].\n\n"
-               "timespec specifies what components of the time to include.\n")},
+               "The optional argument timespec specifies the number of "
+               "additional terms of the time to include.\n"
```

Comment:
Sorry, this is still too long of a line at 95 characters... :(

Doc-sting lines should be wrapped to 72 chars.

(Yes, I realize that this is rather annoying to get right.)

Reply:
Done. A bit of a double standard with the first line of that docstring...